src/Settings.js

```diff
@@ -9,7 +9,12 @@ import { processConfig } from '@duckduckgo/content-scope-scripts/injected/src/ut
  * Some Type helpers to prevent duplication
  * @typedef {import("./deviceApiCalls/__generated__/validators-ts").AutofillFeatureToggles} AutofillFeatureToggles
  * @typedef {import("./deviceApiCalls/__generated__/validators-ts").AvailableInputTypes} AvailableInputTypes
- * @typedef {import("./deviceApiCalls/__generated__/validators-ts").RuntimeConfiguration} RuntimeConfiguration
+ * @typedef {import("./deviceApiCalls/__generated__/validators-ts").BaseRuntimeConfiguration} BaseRuntimeConfiguration
+ * @typedef {{
+ *   contentScope: import("@duckduckgo/privacy-configuration/schema/config").ConfigV4<number>;
+ *   userPreferences: BaseRuntimeConfiguration['userPreferences'];
+ *   userUnprotectedDomains: BaseRuntimeConfiguration['userUnprotectedDomains'];
+ * }} RuntimeConfiguration
  * @typedef {import("../packages/device-api").DeviceApi} DeviceApi
```

Comment:
We can just define RuntimeConfiguration here, using some types from schemas, and some from privacy-configuration repo